papers/michael_lange/michael_lange.rst

```diff
@@ -278,9 +291,13 @@ tutorial by initialising the data associated with the symbolic function
    2\ &\text{for}\ 0.5 \leq x, y \leq 1 \\
    1\ &\text{everywhere else}
 
-The initial condition and the final result after executing the operator
-for 100 timesteps are depicted in Figures :ref:`fig2dconv` and
-:ref:`fig2dconvfinal` respectively.
+The initial condition and the final result after executing the
+operator for 100 timesteps are depicted in Figures :ref:`fig2dconv`
+and :ref:`fig2dconvfinal` respectively. It should be noted that, while
+the results show good agreement with the original tutorial, they do
+exhibit significant numerical errors. This indicates that a high-order
+scheme, as well as a coupled formulation of the advection equation,
+might be preferred to achieve more realistic results.
```

Comment:
...with the original tutorial, they do not represent an accurate solution to the linear convection equation. In particular, the low order spatial discretisation introduces numerical diffusion that changes the shape of the hat into a smooth bump. This is a well-known issue that could be addressed with more sophisticated solver schemes as discussed in [LeVeque, R. J., "Numerical Methods for Conservation Laws", Birkhauser-Verlag (1992)].

_The underlying issue here is that the hat is discontinuous (hence non-differentiable), violating a basic assumption in FD. [The referenced book](https://pdfs.semanticscholar.org/1470/c6f43c769572c4cfc94ffc9c5710484ff1e5.pdf) is discussing this in a very accessible way on pages 8-11. We could avoid the problem by using a smooth initial condition, but then it's not Lorena Barba's tutorial any more._